README.md

```diff
@@ -33,7 +33,7 @@ After running this code, `foo` will be a `Foo` where `foo.array == [1, 3]` and `
 In your Package.swift:
 ```swift
   dependencies: [
-    .package(name: "ResilientDecoding", url: "https://github.com/airbnb/ResilientDecoding.git", from: "0.9.0"),
```

Comment:
Also did we make a mistake and forget to update this before? CocoaPods says 0.9.1 and we also have a 0.9.1 release.

<img width="376" alt="Screen Shot 2020-04-24 at 12 21 51 PM" src="https://user-images.githubusercontent.com/1791049/80249189-40aa4000-8626-11ea-9723-b71250e12be4.png">

Should we clean anything up before moving to the next version?